backend/src/report/report.service.ts

```diff
@@ -339,19 +340,25 @@ export class ReportService {
       values.push(value);
     }
 
-    const defaultHeaderRows = ['Name', 'Notes', 'Schedules'];
+    const defaultHeaderRows = ['ID', 'Name', 'Notes', 'Schedules'];
     const headerRow = values[0]
       ? values[0].map((e) => e.key)
       : defaultHeaderRows;
     const valueRows = values.map((va) => va.map((v) => v.value));
 
-    return XLSX.utils.aoa_to_sheet([headerRow, ...valueRows]);
+    worksheet.addRow(headerRow).font = { bold: true };
```

Comment:
argh 🔥 let's get that **BOLD**

Reply:
very **bold** of you to use another library XD